source/digits_hits/src/GateEnergySpectrumActor.cc

```diff
@@ -226,7 +228,14 @@ void GateEnergySpectrumActor::Construct()
 /// Save data
 void GateEnergySpectrumActor::SaveData()
 {
-    
+   if (mEnableRelativePrimEvents){
+    for(std::list<TH1D*>::iterator it=allEnabledTH1DHistograms.begin();it!=allEnabledTH1DHistograms.end();++it)
+      {
+          (*it)->Scale(1./nEvent);
+      }
+       //pEnergySpectrumFluenceTrack->Scale(1./nEvent);
```

Comment:
Could you remove this dead line of code?

Reply:
done.